editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/maven/CamelProjectConfigurator.java

```diff
@@ -93,9 +101,56 @@ public void configure(ProjectConfigurationRequest request, IProgressMonitor moni
 				// enable the camel nature
 				configureNature(request.getProject(), request.getMavenProject(), monitor);
 			}
+			// handle linked resources for WAR deployments
+			if (isWARProject(request.getProject())) {
+				configureWARStructureMapping(request.getProject());	
+			}		
 		}
 	}
+	
+	private void configureWARStructureMapping(IProject project) throws CoreException {
+		final IVirtualComponent c = ComponentCore.createComponent(project, false);
+		c.create(0, null);
```

Comment:
better to use org.eclipse.wst.common.componentcore.resources.IVirtualResource.NONE instead of 0